### PR TITLE
feat: add devices/list_device_providers endpoint

### PR DESCRIPTION
**Pull request by mc2147**

### CODE REVIEW

File: src/cli/commands/devices.ts [hunk 1]
```diff
@@ -111,6 +111,23 @@ const command: CommandModule<GlobalOptions> = {
...
+          })
+        },
+        async (argv) => {
+          await executeCommand(
+            "devices.listDeviceProviders",
+            [{ provider_category: argv.providerCategory }],
+            argv
+          )
+        }
+      )
```
Comment (codetheweb) [inline src/cli/commands/devices.ts:130]: ❤️ 

the cli hasn't gotten much love recently

**Review by codetheweb — APPROVED** [review]

File: src/cli/commands/devices.ts [hunk 2]
```diff
@@ -111,6 +111,23 @@ const command: CommandModule<GlobalOptions> = {
           await executeCommand("devices.delete", [{ device_id: argv.id }], argv)
         }
       )
+      .command(
+        "list device providers",
```
[the remark this answers is not in the document]
Comment (mc2147) [reply src/cli/commands/devices.ts]: `list_device_providers` sounds good:
`npm run cli:dev devices list_device_providers --provider-category "stable"`

File: src/cli/commands/devices.ts (hunk 1 above)
Comment (mc2147) [reply src/cli/commands/devices.ts:130]: yeah I hope it's working properly lol. I explain in a comment below but when running `cli:dev`, `--*` parameters don't seem to be passed

**Comment by razor-x** [comment]
Maybe the issue is you need to use ` -- `? From the docs https://github.com/seamapi/javascript/pull/171
image

**Comment by mc2147** [comment]
@razor-x 

ugh yep that was it, good catch. Forgot npm run would consume those arguments itself otherwise

**Pull request merged**